packages/utils/src/createErrorHandler.ts

```diff
@@ -0,0 +1,33 @@
+import isPlainObject from 'lodash/isPlainObject';
+
+import { ERRORS_KEY } from './constants';
+import { FieldValidation, FormValidation, GenericObjectType } from './types';
+
+/** Given a `formData` object, recursively creates a `FormValidation` error handling structure around it
+ *
+ * @param formData - The form data around which the error handler is created
+ * @returns - A `FormValidation` object based on the `formData` structure
+ */
+export default function createErrorHandler<T = any>(formData: T): FormValidation<T> {
```

Comment:
@nickgros I am debating whether I rename this to something like `wrapWithErrorHandler()` so that it is an analog to the `unwrap` function.